internal_types/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "internal-types"
+version = "1.0.0" 
+authors = ["Jakub Zajkowski <jakub@casperlabs.io>"]
+edition = "2021"
+description = "Types shared by internal modules of the node. Not intented to be used outside of the node or published as a crate"
+readme = "README.md"
+homepage = "https://casperlabs.io"
+repository = "https://github.com/CasperLabs/casper-node/tree/master/execution_engine_types"
+license = "Apache-2.0"
+
+[dependencies]
+casper-execution-engine = { version = "7.0.1", path = "../execution_engine" }
+casper-types = { version = "5.0.0", path = "../types", features = ["datasize", "json-schema", "std", "std-fs-io"] }
+tracing = { version = "0.1.37", default-features = false }
+rand = { version = "0.8.3", default-features = false, features = ["small_rng"] }
+rand_pcg = { version = "0.3.0", optional = true }
+once_cell = { version = "1.5.2", optional = true }
+datasize = { version = "0.2.11", optional = true }
+serde = { version = "1", features = ["derive", "rc"] }
+
+[features]
+std = ["once_cell"]
+testing = ["rand/default", "rand_pcg"]
+gens = ["testing"]
```

Comment:
I believe "gens" can go and in [tests/Cargo.toml](https://github.com/casper-network/casper-node/pull/4890/files#diff-9218d05dd263a533d0d9c4ebe4224162c35efdb260768ded5cfbc62ffcba1c4b)